packages/kbn-axe-config/index.ts

```diff
@@ -54,5 +54,8 @@ export const AXE_OPTIONS = {
     bypass: {
       enabled: false, // disabled because it's too flaky
     },
+    'nested-interactive': {
+      enabled: false, // disabled because we have too many failures on interactive controls
+    },
```

Comment:
We discussed this change in the A11y Fed sync on 1 March, 2023. I suggested ignoring this rule temporarily so we could gain the max benefit of new rulesets, while we work to remove these nested-interactive issues.